### PR TITLE
GitHub-32555: Updating location to download the CLI

### PR DESCRIPTION
**Pull request by bergerhoffer**
Addresses #32555 and #35079.

Preview: https://deploy-preview-35143--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli_cli-developer-commands

### CODE REVIEW

File: modules/cli-installing-cli.adoc [hunk 1]
```diff
@@ -73,9 +73,9 @@ ifdef::openshift-origin[]
 . Download `oc.tar.gz`.
 endif::[]
 ifndef::openshift-origin[]
-. Navigate to the link:https://cloud.redhat.com/openshift/install[Infrastructure Provider] page on the {cloud-redhat-com} site.
-. Select your infrastructure provider, and, if applicable, your installation type.
-. In the *Command line interface* section, select *Linux* from the drop-down menu and click *Download command-line tools*.
+. Navigate to the link:https://access.redhat.com/downloads/content/290[{product-title} downloads page] on the Red Hat Customer Portal.
+. Select the appropriate version in the *Version* drop-down menu.
```
Comment (zhouying7780) [inline modules/cli-installing-cli.adoc:77]: when I open the 'Red Hat Customer Portal' I saw : Active Subscription Required
 and can't find the 'version' drop-down menu. Is this expected ?

Comment (bergerhoffer) [reply modules/cli-installing-cli.adoc:77]: @zhouying7780 No that's not expected. Are you logged in to the portal (with an account that has subscriptions)?

I assume that you would have had to log in to the old location (in cloud.redhat.com/console.redhat.com) with an account with subscriptions to be able to download the CLI there too.

Comment (bergerhoffer) [reply modules/cli-installing-cli.adoc:77]: @zhouying7780 Let me know if you still see the issue when you are logged in. thanks!

**Pull request merged**